lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -1080,7 +1079,14 @@ SubExprInfo ExprEmitter::visitComb(ICmpOp op) {
 
   auto pred = static_cast<uint64_t>(op.predicate());
   assert(pred < sizeof(symop) / sizeof(symop[0]));
-  if (pred == 1) {
+  if (pred == 0) {
```

Comment:
Please compare against an enum value.  I think they should be defined somewhere, perhaps in a tblgen defined header?

Reply:
Updated with "op.predicate() == ICmpPredicate::eq"

Reply:
nice, thanks!